rtc_tools/rtc_event_log_visualizer/analyzer.cc

```diff
@@ -1380,9 +1380,9 @@ void EventLogAnalyzer::CreateReceiveSideBweSimulationGraph(Plot* plot) {
     }
 
    private:
-    // We don't know the start bitrate, but assume that it is the default 300
+    // We don't know the start bitrate, but assume that it is the default 100
     // kbps.
-    uint32_t last_bitrate_bps_ = 300000;
+    uint32_t last_bitrate_bps_ = 100000;
     bool bitrate_updated_ = false;
   };
 
```

Comment:
There appears to be another one:

https://source.chromium.org/chromium/chromium/src/+/master:third_party/webrtc/pc/peer_connection_factory.cc;l=332;drc=c5f9a81c02375a14b6eea12edd4fbf954ca58ad1;bpv=1;bpt=1